listenbrainz/background/listens_importer/spotify.py

```diff
@@ -30,6 +35,12 @@ def _parse_listen_batch(self, batch: list[dict[str, Any]]) -> list[dict[str, Any
         items = []
         for item in batch:
             try:
+                if (
+                    item["skipped"] or item["incognito_mode"] or
+                    (item["ms_played"] < 30000 and item["reason_end"] in SKIP_REASONS)
```

Comment:
I'm not sure I agree with excluding anything skipped. This isn't what the Spotify live integration does, and is also not what any other scrobble app would do. If the song was listened to long enough it would count as a listen.

Reply:
WRT the spotify importer, I think following their own 30sec threshold is probably better than the (50% or 4min) threshold, at least until we can customize the filtering further.